easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
```

Comment:
not sure if this is really useful, but if you'd like to keep it in, use `http://hpcugent.github.io/easybuild`